package.json

```diff
@@ -104,7 +106,13 @@
   },
   "eik": {
     "server": "https://assets.finn.no",
-    "files": "dist"
+    "type": "package",
+    "files": "eik",
+    "import-map": [
+      "https://assets.finn.no/map/react/v2",
+      "https://assets.finn.no/map/vue/v1",
```

Comment:
It doesn't look like it's relevant for this repo :)

Reply:
True